### PR TITLE
Prepare routes for Connectivity Adapter

### PR DESCRIPTION
**Pull request by pkosiec**
**Description**

Changes proposed in this pull request:

- Prepare routes for Connectivity Adapter

**Related issue(s)**
https://github.com/kyma-incubator/compass/issues/620

### CODE REVIEW

File: components/connectivity-adapter/cmd/main.go [hunk 1]
```diff
@@ -1,16 +1,47 @@
...
-		panic(err)
-	}
+	cfg := config{}
+	err := envconfig.InitWithPrefix(&cfg, "APP")
+	exitOnError(err, "while loading app config")
 
+	router := mux.NewRouter()
+	router.HandleFunc("/healthz", health.HandleFunc)
+
+	appRegistryRouter := router.PathPrefix("/app-registry/v1").Subrouter()
```
Comment (crabtree) [inline components/connectivity-adapter/cmd/main.go]: Shouldn't this be the same as we have the URLs in the application registry and connector? Or do you plan to handle the redirections in a different layer?

I would rather set here the prefix `/v1/metadata`.

File: components/connectivity-adapter/cmd/main.go [hunk 2]
```diff
@@ -1,16 +1,47 @@
...
+	appregistry.RegisterHandler(appRegistryRouter, cfg.AppRegistry)
+
+	connectorRouter := router.PathPrefix("/connector/v1").Subrouter()
```
Comment (crabtree) [inline components/connectivity-adapter/cmd/main.go]: Same as with the `appRegistryRouter` router, currently there is no such route. I would go with `/v1/applications` prefix.

Comment (pkosiec) [reply components/connectivity-adapter/cmd/main.go]: I agree 👍 Fixed

File: components/connectivity-adapter/cmd/main.go (hunk 1 above)
Comment (pkosiec) [reply components/connectivity-adapter/cmd/main.go]: I agree 👍 Fixed

**Pull request merged**